js/thankyou.js

```diff
@@ -28,6 +28,7 @@
 'Terimakasih atas kontribusi anda!',
 'Köszönjük a hozzájárulást!',
 'Takk fyrir þitt framlag!'
```

Comment:
There should be a comma between elements in an array.

Reply:
It's done